### ISSUE TITLE
NameError in MemoryPanel: cnt is not defined

### ISSUE THREAD

**Issue by relekang** [pympler/pympler#21]
Using pympler with debug toolbar raises the following error:

```
Exception Type: NameError at /__debug__/render_panel/
Exception Value: name 'cnt' is not defined
```

**Comment by Pankrat** [comment]
Damn, the memory panel needs better test coverage. I'll commit a fix before the weekend is over. Thanks for the report.

**Comment by Pankrat** [comment]
Should be fixed in 0.4.2.
